src/components/history/History.tsx

```diff
@@ -24,79 +24,92 @@ export default function History(props: HistoryProps) {
 
   const { history, historyProps, serverError } = current.context;
 
-  const historyColumns: Array<Column<HistoryTableEntry>> = React.useMemo(() => [
-    {
-      Header: 'Date',
-      accessor: 'date'
-    },
-    {
-      Header: 'Event Type',
-      accessor: 'eventType'
-    },
-    {
-      Header: 'Work number',
-      accessor: 'workNumber'
-    },
-    {
-      Header: 'User',
-      accessor: 'username'
-    },
-    {
-      Header: 'Source',
-      accessor: 'sourceBarcode',
-      Cell: (props: Cell<HistoryTableEntry>) => {
-        const barcode = props.row.original.sourceBarcode;
-        let style = undefined;
-        if (historyProps.kind === "labwareBarcode" && (barcode === historyProps.value)) {
-          style = { backgroundColor: 'yellow' };
+  const historyColumns: Array<Column<HistoryTableEntry>> = React.useMemo(
+    () => [
+      {
+        Header: 'Date',
+        accessor: 'date'
+      },
+      {
+        Header: 'Event Type',
+        accessor: 'eventType'
+      },
+      {
+        Header: 'Work number',
+        accessor: 'workNumber'
+      },
+      {
+        Header: 'User',
+        accessor: 'username'
+      },
+      {
+        Header: 'Source',
+        accessor: 'sourceBarcode',
+        Cell: (props: Cell<HistoryTableEntry>) => {
+          const barcode = props.row.original.sourceBarcode;
+          let style = undefined;
+          if (historyProps.kind === 'labwareBarcode' && barcode === historyProps.value) {
+            style = { backgroundColor: 'yellow' };
+          }
```

Comment:
Just a thought - may be a 'padding' as well to make the highlight bigger/obvious. I don't know whether it looks good, but may be worth a try

Reply:
Tested with padding and it makes the positioning different a bit different to the non-highlighted barcodes so it looks a bit weird.